keyboards/ergodox_ez/readme.md

```diff
@@ -31,3 +31,59 @@ To flash with ´teensy-loader-cli´:
 
   - Press the Reset button by inserting a paperclip gently into the reset hole
     in the top right corder.
+
+## Settings
+
+You may want to enable QMK_KEYS_PER_SCAN because the Ergodox has a relatively
+slow scan rate. It's been tested with 8 and 16, both work.
+
+## Steno support
+
+Steno support is an experimental-ish feature, mostly useful with
+Plover:
+	https://github.com/openstenoproject/plover/releases
+
+Steno support works by using a virtual serial port and pretending
+to be a hardware steno machine. While steno support uses very large
+chords, the virtual serial form does *not* require NKRO support.
+NKRO is needed only to support sending lots of keys at once over
+USB HID interfaces.
+
+LCDSTENO:
+
+This is a thoroughly experimental feature implemented as a proof
+of concept. This feature assumes you have an LCD display which
+is based on the Hitachi HD44780 or compatible chipset, and are driving
+it through an MCP23008-based I2C adapter, such as AdaFruit's
+i2c/SPI character LCD backpack:
+	https://www.adafruit.com/product/292
+
+The Ergodox is already using I2C for communication with the left
+hand (which uses a closely-related MCP23018 chip). To use this support,
+get the backpack and controller configured. You must set an address
+jumper on the backpack. Set LCDSTENO_PORT to match the address jumper
+you used.
+
+On my Ergodox EZ hardware, the TRRS connectors used for the I2C bus
+are wired as:
+	Sleeve: Ground
+	Ring 1: SCL
+	Ring 2: SDA
+	Ring 3: +5V
+
+IF YOU SCREW THIS UP YOU CAN PROBABLY FRY YOUR HARDWARE. If that sounds
+bad, don't do this. Ask someone else to do it, and then they can pay
+to replace the hardware if they screw it up. See how much better that is?
```

Comment:
IMHO this whole section belongs in your own layout's readme. As does the mod -- i.e, if at all possible, I would duplicate the ergodox.c file into your layout and use that there rather than introduce this hardware mod into the base firmware.